From f3c2abf3f840c6ec0430e56aea362afc774f25ae Mon Sep 17 00:00:00 2001
From: Vladimir 'phcoder' Serbinenko <phcoder@gmail.com>
Date: Thu, 2 May 2013 18:06:39 +0200
Subject: [PATCH 402/482] 	* grub-core/loader/i386/linux.c
 (grub_linux_setup_video): Add 	GRUB_VIDEO_ADAPTER_CAPTURE: to handled drived
 ids.

---
 ChangeLog                     | 5 +++++
 grub-core/loader/i386/linux.c | 1 +
 2 files changed, 6 insertions(+)

diff --git a/ChangeLog b/ChangeLog
index 37595a9..de8acba 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,5 +1,10 @@
 2013-05-02  Vladimir Serbinenko  <phcoder@gmail.com>
 
+	* grub-core/loader/i386/linux.c (grub_linux_setup_video): Add
+	GRUB_VIDEO_ADAPTER_CAPTURE: to handled drived ids.
+
+2013-05-02  Vladimir Serbinenko  <phcoder@gmail.com>
+
 	First automated video test (running videotest and comparing results)
 
 2013-05-02  Vladimir Serbinenko  <phcoder@gmail.com>
diff --git a/grub-core/loader/i386/linux.c b/grub-core/loader/i386/linux.c
index 106496b..4ff6106 100644
--- a/grub-core/loader/i386/linux.c
+++ b/grub-core/loader/i386/linux.c
@@ -343,6 +343,7 @@ grub_linux_setup_video (struct linux_kernel_params *params)
 	  /* Make gcc happy. */
 	case GRUB_VIDEO_DRIVER_SDL:
 	case GRUB_VIDEO_DRIVER_NONE:
+	case GRUB_VIDEO_ADAPTER_CAPTURE:
 	  params->have_vga = GRUB_VIDEO_LINUX_TYPE_SIMPLE;
 	  break;
 	}
-- 
1.8.2.1

